wrangler-core/src/main/java/io/cdap/directives/column/SetType.java

```diff
@@ -131,4 +173,17 @@ public Schema getOutputSchema(SchemaResolutionContext context) {
         .collect(Collectors.toList())
     );
   }
+
+  /**
+   * extracts precision and scale from schema string
+   */
+    public static Pair<Integer, Integer> findPrecisionAndScale(Schema intermediateSchema) {
```

Comment:
Rename to getPrecisionAndScale(Schema schema)

Reply:
Done